tests/FSharp.Data.GraphQL.Tests/FSharp.Data.GraphQL.Tests.fsproj

```diff
@@ -51,5 +50,8 @@
     <ProjectReference Include="..\..\src\FSharp.Data.GraphQL.Server\FSharp.Data.GraphQL.Server.fsproj" />
     <ProjectReference Include="..\..\src\FSharp.Data.GraphQL.Server.Middlewares\FSharp.Data.GraphQL.Server.Middlewares.fsproj" />
   </ItemGroup>
+  <ItemGroup Condition="$(OS) == Unix">
```

Comment:
For some reason, running xUnit tests with `dotnet test` in Unix/OSX machines gives an error in Remoting (reference: https://github.com/xunit/xunit/issues/1357).

Adding a `xunit.runner.json` file with an `"appDomain": "denied"` entry seems to solve this problem as recomended on the issue link.